packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -225,12 +248,20 @@ const RemoteVideoTile = React.memo(
       };
     }, [onDisposeRemoteStreamView, userId]);
 
+    const renderVideoStreamElement = useMemo(() => {
```

Comment:
For readability: Please add a comment here explaining when `renderElement` is not well defined (i.e., about the resource limit) and also that returning `undefined` results in the avatar being shown.

Reply:
will do :)